assets/src/edit-story/components/helpCenter/toggle/index.js

```diff
@@ -127,16 +118,19 @@ function Toggle({
       variant={BUTTON_VARIANTS.RECTANGLE}
       size={BUTTON_SIZES.MEDIUM}
     >
-      <HelpIcon />
-      <Label>{__('Help', 'web-stories')}</Label>
-      {hasNotifications && (
-        <NotificationWrapper>
-          <NotificationBubble notificationCount={notificationCount} />
-        </NotificationWrapper>
-      )}
-      <Icon hasNotifications={hasNotifications} isOpen={isOpen}>
-        <ChevronIcon />
-      </Icon>
+      <Label>{__('Help Center', 'web-stories')}</Label>
+      <Space />
```

Comment:
I like this approach - it also makes RTL simpler, as the divs just go in the other order instead of having to switch margin or padding from right to left. We've used it in a few places too. But I'm okay with changing it.

Reply:
ah, if RTL is the reason then I'm ok with it. I was unaware that this was a standard in the editor.